monitor/service.go

```diff
@@ -316,46 +384,47 @@ func (m *Monitor) storeStatistics() {
 	m.Logger.Printf("Storing statistics in database '%s' retention policy '%s', at interval %s",
 		m.storeDatabase, m.storeRetentionPolicy, m.storeInterval)
 
-	// Get cluster-level metadata. Nothing different is going to happen if errors occur.
-	clusterID := m.MetaClient.ClusterID()
 	hostname, _ := os.Hostname()
-	clusterTags := map[string]string{
-		"clusterID": fmt.Sprintf("%d", clusterID),
-		"nodeID":    fmt.Sprintf("%d", m.NodeID),
-		"hostname":  hostname,
-	}
+	m.SetGlobalTag("hostname", hostname)
 
+	m.mu.Lock()
 	tick := time.NewTicker(m.storeInterval)
+	m.mu.Unlock()
```

Comment:
Is `m.storeInterval` guaranteed to be greater than zero? A `time.NewTicker(0)` will panic which can leave your mutex locked (if the panic is recovered).